modules/ROOT/pages/app-dashboards.adoc

```diff
@@ -31,7 +31,7 @@ image::builtin-dashboard-selection-interaction.png[Built-in Dashboard Selector]
 
 Your data is pre-populated and displayed in the built-in dashboards and logs, so no configuration is needed.
 
-Built-in dashboards include various pages to view different types of charts. 
+Built-in dashboards include various pages to view different types of charts.
```

Comment:
```suggestion
Built-in dashboards include various pages to view different types of charts:
```
End with a colon so that the audience knows it will learn more about "various" and "different types."

Reply:
BUT...this is WAY too much information for inclusion in a table. A table is intended to be a scannable matrix of related information. Cell entries that are any longer than a sentence or two disqualifies table format.

I will COPYEDIT this as it exists, but I strongly advise that you reorganize this as follows:

```suggestion
Built-in dashboards include the following pages, which in turn are populated by different types of charts:
* Overview
* Inbound
* Outbound
* ....
```
Then make each page its own subtopic, listing and describing the charts.

Table cells should never include sublists, notations, examples, or cross-references. Those are red flags that point you to finding an alternative presentation method.

Reply:
I agree with you; I will make a JIRA to reorganize this. Alternatively, are footnotes a thing?